{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[],"authorship_tag":"ABX9TyMBpqBpKZgNFNgr7h8FcaEF"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"code","source":["from google.colab import drive\n","drive.mount('/content/drive', force_remount=True)\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"iV9oZ2IWQN9F","executionInfo":{"status":"ok","timestamp":1763222657432,"user_tz":300,"elapsed":69162,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"}},"outputId":"81303339-8bcd-4264-e7fa-38e8b1bcff09"},"execution_count":9,"outputs":[{"output_type":"stream","name":"stdout","text":["Mounted at /content/drive\n"]}]},{"cell_type":"code","source":["!mkdir -p /content/CTC360/{fastq,trimmed,qc,bam,variants,figures,ref}\n","project_dir = \"/content/CTC360\"\n","print(\"Project directory created.\")\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"1MDYwz6eSlyG","executionInfo":{"status":"ok","timestamp":1763223181244,"user_tz":300,"elapsed":108,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"}},"outputId":"b940bb80-9c54-46b0-8b6e-15535340c4ed"},"execution_count":10,"outputs":[{"output_type":"stream","name":"stdout","text":["Project directory created.\n"]}]},{"cell_type":"code","source":["!apt-get update\n","!apt-get install -y sra-toolkit fastqc bwa samtools fastp\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"id":"UAPrDtbJSqGZ","executionInfo":{"status":"ok","timestamp":1763223241844,"user_tz":300,"elapsed":31333,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"}},"outputId":"51ffcaf3-e56d-46d9-c68e-568c65d174e7"},"execution_count":11,"outputs":[{"output_type":"stream","name":"stdout","text":["Get:1 https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/ InRelease [3,632 B]\n","Get:2 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  InRelease [1,581 B]\n","Get:3 https://cli.github.com/packages stable InRelease [3,917 B]\n","Hit:4 http://archive.ubuntu.com/ubuntu jammy InRelease\n","Get:5 http://security.ubuntu.com/ubuntu jammy-security InRelease [129 kB]\n","Get:6 https://r2u.stat.illinois.edu/ubuntu jammy InRelease [6,555 B]\n","Get:7 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [128 kB]\n","Hit:8 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease\n","Get:9 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  Packages [2,143 kB]\n","Hit:10 https://ppa.launchpadcontent.net/graphics-drivers/ppa/ubuntu jammy InRelease\n","Get:11 https://cli.github.com/packages stable/main amd64 Packages [343 B]\n","Hit:12 https://ppa.launchpadcontent.net/ubuntugis/ppa/ubuntu jammy InRelease\n","Get:13 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [127 kB]\n","Get:14 https://r2u.stat.illinois.edu/ubuntu jammy/main all Packages [9,452 kB]\n","Get:15 https://r2u.stat.illinois.edu/ubuntu jammy/main amd64 Packages [2,827 kB]\n","Get:16 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1,595 kB]\n","Get:17 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [3,861 kB]\n","Get:18 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [6,181 kB]\n","Fetched 26.5 MB in 3s (8,093 kB/s)\n","Reading package lists... Done\n","W: Skipping acquire of configured file 'main/source/Sources' as repository 'https://r2u.stat.illinois.edu/ubuntu jammy InRelease' does not seem to provide it (sources.list entry misspelt?)\n","Reading package lists... Done\n","Building dependency tree... Done\n","Reading state information... Done\n","The following additional packages will be installed:\n","  at-spi2-core blends-common default-jre default-jre-headless\n","  fonts-dejavu-core fonts-dejavu-extra gsettings-desktop-schemas\n","  libapache-pom-java libargs4j-java libatk-bridge2.0-0 libatk-wrapper-java\n","  libatk-wrapper-java-jni libatk1.0-0 libatk1.0-data libatspi2.0-0\n","  libcommons-compress-java libcommons-io-java libcommons-jexl2-java\n","  libcommons-lang3-java libcommons-logging-java libcommons-math3-java\n","  libcommons-parent-java libfindbin-libs-perl libhts3 libhtscodecs2\n","  libhtsjdk-java libjbzip2-java libjson-simple-java libkdf5-2 libncbi-vdb2\n","  libncbi-wvdb2 libngs-java libngs-sdk-dev libngs-sdk2 libsis-base-java\n","  libsis-base-jni libsis-jhdf5-java libsis-jhdf5-jni libsnappy-java\n","  libsnappy-jni libxcomposite1 libxtst6 libxxf86dga1 med-config menu\n","  openjdk-11-jre openjdk-11-jre-headless session-migration x11-utils\n","Suggested packages:\n","  blends-doc libxz-java libcommons-io-java-doc libcommons-lang3-java-doc\n","  libavalon-framework-java libcommons-logging-java-doc\n","  libexcalibur-logkit-java liblog4j1.2-java picard-tools libjson-simple-doc\n","  menu-l10n gksu | kde-runtime | ktsuss libnss-mdns fonts-ipafont-gothic\n","  fonts-ipafont-mincho fonts-wqy-microhei | fonts-wqy-zenhei fonts-indic\n","  cwltool mesa-utils\n","The following NEW packages will be installed:\n","  at-spi2-core blends-common bwa default-jre default-jre-headless fastp fastqc\n","  fonts-dejavu-core fonts-dejavu-extra gsettings-desktop-schemas\n","  libapache-pom-java libargs4j-java libatk-bridge2.0-0 libatk-wrapper-java\n","  libatk-wrapper-java-jni libatk1.0-0 libatk1.0-data libatspi2.0-0\n","  libcommons-compress-java libcommons-io-java libcommons-jexl2-java\n","  libcommons-lang3-java libcommons-logging-java libcommons-math3-java\n","  libcommons-parent-java libfindbin-libs-perl libhts3 libhtscodecs2\n","  libhtsjdk-java libjbzip2-java libjson-simple-java libkdf5-2 libncbi-vdb2\n","  libncbi-wvdb2 libngs-java libngs-sdk-dev libngs-sdk2 libsis-base-java\n","  libsis-base-jni libsis-jhdf5-java libsis-jhdf5-jni libsnappy-java\n","  libsnappy-jni libxcomposite1 libxtst6 libxxf86dga1 med-config menu\n","  openjdk-11-jre openjdk-11-jre-headless samtools session-migration\n","  sra-toolkit x11-utils\n","0 upgraded, 54 newly installed, 0 to remove and 48 not upgraded.\n","Need to get 64.2 MB of archives.\n","After this operation, 229 MB of additional disk space will be used.\n","Get:1 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatspi2.0-0 amd64 2.44.0-3 [80.9 kB]\n","Get:2 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxtst6 amd64 2:1.2.3-1build4 [13.4 kB]\n","Get:3 http://archive.ubuntu.com/ubuntu jammy/main amd64 session-migration amd64 0.3.6 [9,774 B]\n","Get:4 http://archive.ubuntu.com/ubuntu jammy/main amd64 gsettings-desktop-schemas all 42.0-1ubuntu1 [31.1 kB]\n","Get:5 http://archive.ubuntu.com/ubuntu jammy/main amd64 at-spi2-core amd64 2.44.0-3 [54.4 kB]\n","Get:6 http://archive.ubuntu.com/ubuntu jammy/universe amd64 menu amd64 2.1.47ubuntu4 [354 kB]\n","Get:7 http://archive.ubuntu.com/ubuntu jammy/universe amd64 blends-common all 0.7.4ubuntu1 [15.9 kB]\n","Get:8 http://archive.ubuntu.com/ubuntu jammy/universe amd64 bwa amd64 0.7.17-6 [195 kB]\n","Get:9 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 openjdk-11-jre-headless amd64 11.0.28+6-1ubuntu1~22.04.1 [42.6 MB]\n","Get:10 http://archive.ubuntu.com/ubuntu jammy/main amd64 default-jre-headless amd64 2:1.11-72build2 [3,042 B]\n","Get:11 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 openjdk-11-jre amd64 11.0.28+6-1ubuntu1~22.04.1 [214 kB]\n","Get:12 http://archive.ubuntu.com/ubuntu jammy/main amd64 default-jre amd64 2:1.11-72build2 [896 B]\n","Get:13 http://archive.ubuntu.com/ubuntu jammy/universe amd64 fastp amd64 0.20.1+dfsg-1 [193 kB]\n","Get:14 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-math3-java all 3.6.1-3 [2,016 kB]\n","Get:15 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libjbzip2-java all 0.9.1-7 [42.8 kB]\n","Get:16 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libapache-pom-java all 18-1 [4,720 B]\n","Get:17 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-parent-java all 43-1 [10.8 kB]\n","Get:18 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-compress-java all 1.21-1 [575 kB]\n","Get:19 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-logging-java all 1.2-2 [60.3 kB]\n","Get:20 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-jexl2-java all 2.1.1-5 [251 kB]\n","Get:21 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libjson-simple-java all 2.3.0-1 [61.0 kB]\n","Get:22 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libngs-sdk2 amd64 2.11.2-3 [65.6 kB]\n","Get:23 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libngs-sdk-dev amd64 2.11.2-3 [111 kB]\n","Get:24 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libngs-java amd64 2.11.2-3 [75.8 kB]\n","Get:25 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsnappy-jni amd64 1.1.8.3-1 [6,756 B]\n","Get:26 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsnappy-java all 1.1.8.3-1 [93.9 kB]\n","Get:27 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhtsjdk-java all 2.24.1+dfsg-1 [1,644 kB]\n","Get:28 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libfindbin-libs-perl all 2.190.02-1 [23.2 kB]\n","Get:29 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libargs4j-java all 2.33-1.1 [137 kB]\n","Get:30 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-io-java all 2.11.0-2 [297 kB]\n","Get:31 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libcommons-lang3-java all 3.11-1 [526 kB]\n","Get:32 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsis-base-jni amd64 18.09~pre1+git20180928.45fbd31+dfsg-3 [12.7 kB]\n","Get:33 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsis-base-java all 18.09~pre1+git20180928.45fbd31+dfsg-3 [96.4 kB]\n","Get:34 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsis-jhdf5-jni amd64 19.04.0+dfsg-4 [96.6 kB]\n","Get:35 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libsis-jhdf5-java all 19.04.0+dfsg-4 [1,490 kB]\n","Get:36 http://archive.ubuntu.com/ubuntu jammy/universe amd64 fastqc all 0.11.9+dfsg-5 [370 kB]\n","Get:37 http://archive.ubuntu.com/ubuntu jammy/main amd64 fonts-dejavu-core all 2.37-2build1 [1,041 kB]\n","Get:38 http://archive.ubuntu.com/ubuntu jammy/main amd64 fonts-dejavu-extra all 2.37-2build1 [2,041 kB]\n","Get:39 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk1.0-data all 2.36.0-3build1 [2,824 B]\n","Get:40 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk1.0-0 amd64 2.36.0-3build1 [51.9 kB]\n","Get:41 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk-bridge2.0-0 amd64 2.38.0-3 [66.6 kB]\n","Get:42 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxcomposite1 amd64 1:0.4.5-1build2 [7,192 B]\n","Get:43 http://archive.ubuntu.com/ubuntu jammy/main amd64 libxxf86dga1 amd64 2:1.1.5-0ubuntu3 [12.6 kB]\n","Get:44 http://archive.ubuntu.com/ubuntu jammy/main amd64 x11-utils amd64 7.7+5build2 [206 kB]\n","Get:45 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk-wrapper-java all 0.38.0-5build1 [53.1 kB]\n","Get:46 http://archive.ubuntu.com/ubuntu jammy/main amd64 libatk-wrapper-java-jni amd64 0.38.0-5build1 [49.0 kB]\n","Get:47 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhtscodecs2 amd64 1.1.1-3 [53.2 kB]\n","Get:48 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhts3 amd64 1.13+ds-2build1 [390 kB]\n","Get:49 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libkdf5-2 amd64 2.11.2+dfsg-4build2 [14.7 kB]\n","Get:50 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libncbi-vdb2 amd64 2.11.2+dfsg-4build2 [1,364 kB]\n","Get:51 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libncbi-wvdb2 amd64 2.11.2+dfsg-4build2 [1,252 kB]\n","Get:52 http://archive.ubuntu.com/ubuntu jammy/universe amd64 samtools amd64 1.13-4 [520 kB]\n","Get:53 http://archive.ubuntu.com/ubuntu jammy/universe amd64 sra-toolkit amd64 2.11.3+dfsg-1ubuntu1 [5,277 kB]\n","Get:54 http://archive.ubuntu.com/ubuntu jammy/universe amd64 med-config all 3.7.1 [11.6 kB]\n","Fetched 64.2 MB in 3s (22.6 MB/s)\n","Extracting templates from packages: 100%\n","Preconfiguring packages ...\n","Selecting previously unselected package libatspi2.0-0:amd64.\n","(Reading database ... 121703 files and directories currently installed.)\n","Preparing to unpack .../00-libatspi2.0-0_2.44.0-3_amd64.deb ...\n","Unpacking libatspi2.0-0:amd64 (2.44.0-3) ...\n","Selecting previously unselected package libxtst6:amd64.\n","Preparing to unpack .../01-libxtst6_2%3a1.2.3-1build4_amd64.deb ...\n","Unpacking libxtst6:amd64 (2:1.2.3-1build4) ...\n","Selecting previously unselected package session-migration.\n","Preparing to unpack .../02-session-migration_0.3.6_amd64.deb ...\n","Unpacking session-migration (0.3.6) ...\n","Selecting previously unselected package gsettings-desktop-schemas.\n","Preparing to unpack .../03-gsettings-desktop-schemas_42.0-1ubuntu1_all.deb ...\n","Unpacking gsettings-desktop-schemas (42.0-1ubuntu1) ...\n","Selecting previously unselected package at-spi2-core.\n","Preparing to unpack .../04-at-spi2-core_2.44.0-3_amd64.deb ...\n","Unpacking at-spi2-core (2.44.0-3) ...\n","Selecting previously unselected package menu.\n","Preparing to unpack .../05-menu_2.1.47ubuntu4_amd64.deb ...\n","Unpacking menu (2.1.47ubuntu4) ...\n","Selecting previously unselected package blends-common.\n","Preparing to unpack .../06-blends-common_0.7.4ubuntu1_all.deb ...\n","Unpacking blends-common (0.7.4ubuntu1) ...\n","Selecting previously unselected package bwa.\n","Preparing to unpack .../07-bwa_0.7.17-6_amd64.deb ...\n","Unpacking bwa (0.7.17-6) ...\n","Selecting previously unselected package openjdk-11-jre-headless:amd64.\n","Preparing to unpack .../08-openjdk-11-jre-headless_11.0.28+6-1ubuntu1~22.04.1_amd64.deb ...\n","Unpacking openjdk-11-jre-headless:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n","Selecting previously unselected package default-jre-headless.\n","Preparing to unpack .../09-default-jre-headless_2%3a1.11-72build2_amd64.deb ...\n","Unpacking default-jre-headless (2:1.11-72build2) ...\n","Selecting previously unselected package openjdk-11-jre:amd64.\n","Preparing to unpack .../10-openjdk-11-jre_11.0.28+6-1ubuntu1~22.04.1_amd64.deb ...\n","Unpacking openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n","Selecting previously unselected package default-jre.\n","Preparing to unpack .../11-default-jre_2%3a1.11-72build2_amd64.deb ...\n","Unpacking default-jre (2:1.11-72build2) ...\n","Selecting previously unselected package fastp.\n","Preparing to unpack .../12-fastp_0.20.1+dfsg-1_amd64.deb ...\n","Unpacking fastp (0.20.1+dfsg-1) ...\n","Selecting previously unselected package libcommons-math3-java.\n","Preparing to unpack .../13-libcommons-math3-java_3.6.1-3_all.deb ...\n","Unpacking libcommons-math3-java (3.6.1-3) ...\n","Selecting previously unselected package libjbzip2-java.\n","Preparing to unpack .../14-libjbzip2-java_0.9.1-7_all.deb ...\n","Unpacking libjbzip2-java (0.9.1-7) ...\n","Selecting previously unselected package libapache-pom-java.\n","Preparing to unpack .../15-libapache-pom-java_18-1_all.deb ...\n","Unpacking libapache-pom-java (18-1) ...\n","Selecting previously unselected package libcommons-parent-java.\n","Preparing to unpack .../16-libcommons-parent-java_43-1_all.deb ...\n","Unpacking libcommons-parent-java (43-1) ...\n","Selecting previously unselected package libcommons-compress-java.\n","Preparing to unpack .../17-libcommons-compress-java_1.21-1_all.deb ...\n","Unpacking libcommons-compress-java (1.21-1) ...\n","Selecting previously unselected package libcommons-logging-java.\n","Preparing to unpack .../18-libcommons-logging-java_1.2-2_all.deb ...\n","Unpacking libcommons-logging-java (1.2-2) ...\n","Selecting previously unselected package libcommons-jexl2-java.\n","Preparing to unpack .../19-libcommons-jexl2-java_2.1.1-5_all.deb ...\n","Unpacking libcommons-jexl2-java (2.1.1-5) ...\n","Selecting previously unselected package libjson-simple-java.\n","Preparing to unpack .../20-libjson-simple-java_2.3.0-1_all.deb ...\n","Unpacking libjson-simple-java (2.3.0-1) ...\n","Selecting previously unselected package libngs-sdk2:amd64.\n","Preparing to unpack .../21-libngs-sdk2_2.11.2-3_amd64.deb ...\n","Unpacking libngs-sdk2:amd64 (2.11.2-3) ...\n","Selecting previously unselected package libngs-sdk-dev:amd64.\n","Preparing to unpack .../22-libngs-sdk-dev_2.11.2-3_amd64.deb ...\n","Unpacking libngs-sdk-dev:amd64 (2.11.2-3) ...\n","Selecting previously unselected package libngs-java:amd64.\n","Preparing to unpack .../23-libngs-java_2.11.2-3_amd64.deb ...\n","Unpacking libngs-java:amd64 (2.11.2-3) ...\n","Selecting previously unselected package libsnappy-jni.\n","Preparing to unpack .../24-libsnappy-jni_1.1.8.3-1_amd64.deb ...\n","Unpacking libsnappy-jni (1.1.8.3-1) ...\n","Selecting previously unselected package libsnappy-java.\n","Preparing to unpack .../25-libsnappy-java_1.1.8.3-1_all.deb ...\n","Unpacking libsnappy-java (1.1.8.3-1) ...\n","Selecting previously unselected package libhtsjdk-java.\n","Preparing to unpack .../26-libhtsjdk-java_2.24.1+dfsg-1_all.deb ...\n","Unpacking libhtsjdk-java (2.24.1+dfsg-1) ...\n","Selecting previously unselected package libfindbin-libs-perl.\n","Preparing to unpack .../27-libfindbin-libs-perl_2.190.02-1_all.deb ...\n","Unpacking libfindbin-libs-perl (2.190.02-1) ...\n","Selecting previously unselected package libargs4j-java.\n","Preparing to unpack .../28-libargs4j-java_2.33-1.1_all.deb ...\n","Unpacking libargs4j-java (2.33-1.1) ...\n","Selecting previously unselected package libcommons-io-java.\n","Preparing to unpack .../29-libcommons-io-java_2.11.0-2_all.deb ...\n","Unpacking libcommons-io-java (2.11.0-2) ...\n","Selecting previously unselected package libcommons-lang3-java.\n","Preparing to unpack .../30-libcommons-lang3-java_3.11-1_all.deb ...\n","Unpacking libcommons-lang3-java (3.11-1) ...\n","Selecting previously unselected package libsis-base-jni.\n","Preparing to unpack .../31-libsis-base-jni_18.09~pre1+git20180928.45fbd31+dfsg-3_amd64.deb ...\n","Unpacking libsis-base-jni (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n","Selecting previously unselected package libsis-base-java.\n","Preparing to unpack .../32-libsis-base-java_18.09~pre1+git20180928.45fbd31+dfsg-3_all.deb ...\n","Unpacking libsis-base-java (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n","Selecting previously unselected package libsis-jhdf5-jni.\n","Preparing to unpack .../33-libsis-jhdf5-jni_19.04.0+dfsg-4_amd64.deb ...\n","Unpacking libsis-jhdf5-jni (19.04.0+dfsg-4) ...\n","Selecting previously unselected package libsis-jhdf5-java.\n","Preparing to unpack .../34-libsis-jhdf5-java_19.04.0+dfsg-4_all.deb ...\n","Unpacking libsis-jhdf5-java (19.04.0+dfsg-4) ...\n","Selecting previously unselected package fastqc.\n","Preparing to unpack .../35-fastqc_0.11.9+dfsg-5_all.deb ...\n","Unpacking fastqc (0.11.9+dfsg-5) ...\n","Selecting previously unselected package fonts-dejavu-core.\n","Preparing to unpack .../36-fonts-dejavu-core_2.37-2build1_all.deb ...\n","Unpacking fonts-dejavu-core (2.37-2build1) ...\n","Selecting previously unselected package fonts-dejavu-extra.\n","Preparing to unpack .../37-fonts-dejavu-extra_2.37-2build1_all.deb ...\n","Unpacking fonts-dejavu-extra (2.37-2build1) ...\n","Selecting previously unselected package libatk1.0-data.\n","Preparing to unpack .../38-libatk1.0-data_2.36.0-3build1_all.deb ...\n","Unpacking libatk1.0-data (2.36.0-3build1) ...\n","Selecting previously unselected package libatk1.0-0:amd64.\n","Preparing to unpack .../39-libatk1.0-0_2.36.0-3build1_amd64.deb ...\n","Unpacking libatk1.0-0:amd64 (2.36.0-3build1) ...\n","Selecting previously unselected package libatk-bridge2.0-0:amd64.\n","Preparing to unpack .../40-libatk-bridge2.0-0_2.38.0-3_amd64.deb ...\n","Unpacking libatk-bridge2.0-0:amd64 (2.38.0-3) ...\n","Selecting previously unselected package libxcomposite1:amd64.\n","Preparing to unpack .../41-libxcomposite1_1%3a0.4.5-1build2_amd64.deb ...\n","Unpacking libxcomposite1:amd64 (1:0.4.5-1build2) ...\n","Selecting previously unselected package libxxf86dga1:amd64.\n","Preparing to unpack .../42-libxxf86dga1_2%3a1.1.5-0ubuntu3_amd64.deb ...\n","Unpacking libxxf86dga1:amd64 (2:1.1.5-0ubuntu3) ...\n","Selecting previously unselected package x11-utils.\n","Preparing to unpack .../43-x11-utils_7.7+5build2_amd64.deb ...\n","Unpacking x11-utils (7.7+5build2) ...\n","Selecting previously unselected package libatk-wrapper-java.\n","Preparing to unpack .../44-libatk-wrapper-java_0.38.0-5build1_all.deb ...\n","Unpacking libatk-wrapper-java (0.38.0-5build1) ...\n","Selecting previously unselected package libatk-wrapper-java-jni:amd64.\n","Preparing to unpack .../45-libatk-wrapper-java-jni_0.38.0-5build1_amd64.deb ...\n","Unpacking libatk-wrapper-java-jni:amd64 (0.38.0-5build1) ...\n","Selecting previously unselected package libhtscodecs2:amd64.\n","Preparing to unpack .../46-libhtscodecs2_1.1.1-3_amd64.deb ...\n","Unpacking libhtscodecs2:amd64 (1.1.1-3) ...\n","Selecting previously unselected package libhts3:amd64.\n","Preparing to unpack .../47-libhts3_1.13+ds-2build1_amd64.deb ...\n","Unpacking libhts3:amd64 (1.13+ds-2build1) ...\n","Selecting previously unselected package libkdf5-2.\n","Preparing to unpack .../48-libkdf5-2_2.11.2+dfsg-4build2_amd64.deb ...\n","Unpacking libkdf5-2 (2.11.2+dfsg-4build2) ...\n","Selecting previously unselected package libncbi-vdb2.\n","Preparing to unpack .../49-libncbi-vdb2_2.11.2+dfsg-4build2_amd64.deb ...\n","Unpacking libncbi-vdb2 (2.11.2+dfsg-4build2) ...\n","Selecting previously unselected package libncbi-wvdb2.\n","Preparing to unpack .../50-libncbi-wvdb2_2.11.2+dfsg-4build2_amd64.deb ...\n","Unpacking libncbi-wvdb2 (2.11.2+dfsg-4build2) ...\n","Selecting previously unselected package samtools.\n","Preparing to unpack .../51-samtools_1.13-4_amd64.deb ...\n","Unpacking samtools (1.13-4) ...\n","Selecting previously unselected package sra-toolkit.\n","Preparing to unpack .../52-sra-toolkit_2.11.3+dfsg-1ubuntu1_amd64.deb ...\n","Unpacking sra-toolkit (2.11.3+dfsg-1ubuntu1) ...\n","Selecting previously unselected package med-config.\n","Preparing to unpack .../53-med-config_3.7.1_all.deb ...\n","Unpacking med-config (3.7.1) ...\n","Setting up libhtscodecs2:amd64 (1.1.1-3) ...\n","Setting up session-migration (0.3.6) ...\n","Created symlink /etc/systemd/user/graphical-session-pre.target.wants/session-migration.service → /usr/lib/systemd/user/session-migration.service.\n","Setting up libhts3:amd64 (1.13+ds-2build1) ...\n","Setting up bwa (0.7.17-6) ...\n","Setting up libxtst6:amd64 (2:1.2.3-1build4) ...\n","Setting up openjdk-11-jre-headless:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n","update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/jjs to provide /usr/bin/jjs (jjs) in auto mode\n","update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/rmid to provide /usr/bin/rmid (rmid) in auto mode\n","update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/pack200 to provide /usr/bin/pack200 (pack200) in auto mode\n","update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/unpack200 to provide /usr/bin/unpack200 (unpack200) in auto mode\n","Setting up libngs-sdk2:amd64 (2.11.2-3) ...\n","Setting up libxxf86dga1:amd64 (2:1.1.5-0ubuntu3) ...\n","Setting up openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n","Setting up libfindbin-libs-perl (2.190.02-1) ...\n","Setting up libncbi-vdb2 (2.11.2+dfsg-4build2) ...\n","Setting up libatspi2.0-0:amd64 (2.44.0-3) ...\n","Setting up libjson-simple-java (2.3.0-1) ...\n","Setting up libkdf5-2 (2.11.2+dfsg-4build2) ...\n","Setting up samtools (1.13-4) ...\n","Setting up fastp (0.20.1+dfsg-1) ...\n","Setting up libsis-base-jni (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n","Setting up libncbi-wvdb2 (2.11.2+dfsg-4build2) ...\n","Setting up libapache-pom-java (18-1) ...\n","Setting up fonts-dejavu-core (2.37-2build1) ...\n","Setting up libjbzip2-java (0.9.1-7) ...\n","Setting up libngs-sdk-dev:amd64 (2.11.2-3) ...\n","Setting up libatk1.0-data (2.36.0-3build1) ...\n","Setting up fonts-dejavu-extra (2.37-2build1) ...\n","Setting up libcommons-math3-java (3.6.1-3) ...\n","Setting up libsis-jhdf5-jni (19.04.0+dfsg-4) ...\n","Setting up libatk1.0-0:amd64 (2.36.0-3build1) ...\n","Setting up libngs-java:amd64 (2.11.2-3) ...\n","Setting up menu (2.1.47ubuntu4) ...\n","Setting up libsnappy-jni (1.1.8.3-1) ...\n","Setting up libxcomposite1:amd64 (1:0.4.5-1build2) ...\n","Setting up libargs4j-java (2.33-1.1) ...\n","Setting up blends-common (0.7.4ubuntu1) ...\n","Setting up gsettings-desktop-schemas (42.0-1ubuntu1) ...\n","Setting up sra-toolkit (2.11.3+dfsg-1ubuntu1) ...\n","Setting up med-config (3.7.1) ...\n","Adding group `med' (GID 107) ...\n","Done.\n","Setting up default-jre-headless (2:1.11-72build2) ...\n","Setting up libcommons-parent-java (43-1) ...\n","Setting up libcommons-logging-java (1.2-2) ...\n","Setting up default-jre (2:1.11-72build2) ...\n","Setting up libatk-bridge2.0-0:amd64 (2.38.0-3) ...\n","Setting up libcommons-jexl2-java (2.1.1-5) ...\n","Setting up libcommons-lang3-java (3.11-1) ...\n","Setting up x11-utils (7.7+5build2) ...\n","Setting up libsnappy-java (1.1.8.3-1) ...\n","Setting up libatk-wrapper-java (0.38.0-5build1) ...\n","Setting up libcommons-compress-java (1.21-1) ...\n","Setting up libcommons-io-java (2.11.0-2) ...\n","Setting up libatk-wrapper-java-jni:amd64 (0.38.0-5build1) ...\n","Setting up libsis-base-java (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n","Setting up libhtsjdk-java (2.24.1+dfsg-1) ...\n","Setting up libsis-jhdf5-java (19.04.0+dfsg-4) ...\n","Setting up fastqc (0.11.9+dfsg-5) ...\n","Processing triggers for mailcap (3.70+nmu1ubuntu1) ...\n","Processing triggers for fontconfig (2.13.1-4.2ubuntu5) ...\n","Processing triggers for hicolor-icon-theme (0.17-2) ...\n","Processing triggers for libglib2.0-0:amd64 (2.72.4-0ubuntu2.6) ...\n","Processing triggers for libc-bin (2.35-0ubuntu3.8) ...\n","/sbin/ldconfig.real: /usr/local/lib/libur_adapter_opencl.so.0 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtcm_debug.so.1 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtcm.so.1 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libumf.so.1 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero_v2.so.0 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libur_loader.so.0 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero.so.0 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libhwloc.so.15 is not a symbolic link\n","\n","Processing triggers for man-db (2.10.2-1) ...\n","Setting up at-spi2-core (2.44.0-3) ...\n","Processing triggers for menu (2.1.47ubuntu4) ...\n"]}]},{"cell_type":"code","source":["!cd /content\n","!wget -c https://github.com/broadinstitute/gatk/releases/download/4.4.0.0/gatk-4.4.0.0.zip\n","!unzip -o gatk-4.4.0.0.zip\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"id":"6pKKpoJQTOZk","executionInfo":{"status":"ok","timestamp":1763223352384,"user_tz":300,"elapsed":14974,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"}},"outputId":"337c3e8e-1cf0-4f58-e0e7-cbb6748203e0"},"execution_count":13,"outputs":[{"output_type":"stream","name":"stdout","text":["--2025-11-15 16:15:36--  https://github.com/broadinstitute/gatk/releases/download/4.4.0.0/gatk-4.4.0.0.zip\n","Resolving github.com (github.com)... 140.82.113.4\n","Connecting to github.com (github.com)|140.82.113.4|:443... connected.\n","HTTP request sent, awaiting response... 302 Found\n","Location: https://release-assets.githubusercontent.com/github-production-release-asset/27452807/0bd42ce9-3dc6-4814-971c-264ebaf66d3d?sp=r&sv=2018-11-09&sr=b&spr=https&se=2025-11-15T17%3A15%3A40Z&rscd=attachment%3B+filename%3Dgatk-4.4.0.0.zip&rsct=application%2Foctet-stream&skoid=96c2d410-5711-43a1-aedd-ab1947aa7ab0&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skt=2025-11-15T16%3A15%3A37Z&ske=2025-11-15T17%3A15%3A40Z&sks=b&skv=2018-11-09&sig=F9fY%2FLEaVBEUvjPzODMwCMesFcBvIMnkzOjV74dhJNs%3D&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmVsZWFzZS1hc3NldHMuZ2l0aHVidXNlcmNvbnRlbnQuY29tIiwia2V5Ijoia2V5MSIsImV4cCI6MTc2MzIyNjkzNywibmJmIjoxNzYzMjIzMzM3LCJwYXRoIjoicmVsZWFzZWFzc2V0cHJvZHVjdGlvbi5ibG9iLmNvcmUud2luZG93cy5uZXQifQ.KzIcTHG91lu0ambiXqS2gVklpB7nrFOmUT3G-VvKLls&response-content-disposition=attachment%3B%20filename%3Dgatk-4.4.0.0.zip&response-content-type=application%2Foctet-stream [following]\n","--2025-11-15 16:15:37--  https://release-assets.githubusercontent.com/github-production-release-asset/27452807/0bd42ce9-3dc6-4814-971c-264ebaf66d3d?sp=r&sv=2018-11-09&sr=b&spr=https&se=2025-11-15T17%3A15%3A40Z&rscd=attachment%3B+filename%3Dgatk-4.4.0.0.zip&rsct=application%2Foctet-stream&skoid=96c2d410-5711-43a1-aedd-ab1947aa7ab0&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skt=2025-11-15T16%3A15%3A37Z&ske=2025-11-15T17%3A15%3A40Z&sks=b&skv=2018-11-09&sig=F9fY%2FLEaVBEUvjPzODMwCMesFcBvIMnkzOjV74dhJNs%3D&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmVsZWFzZS1hc3NldHMuZ2l0aHVidXNlcmNvbnRlbnQuY29tIiwia2V5Ijoia2V5MSIsImV4cCI6MTc2MzIyNjkzNywibmJmIjoxNzYzMjIzMzM3LCJwYXRoIjoicmVsZWFzZWFzc2V0cHJvZHVjdGlvbi5ibG9iLmNvcmUud2luZG93cy5uZXQifQ.KzIcTHG91lu0ambiXqS2gVklpB7nrFOmUT3G-VvKLls&response-content-disposition=attachment%3B%20filename%3Dgatk-4.4.0.0.zip&response-content-type=application%2Foctet-stream\n","Resolving release-assets.githubusercontent.com (release-assets.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n","Connecting to release-assets.githubusercontent.com (release-assets.githubusercontent.com)|185.199.108.133|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 642466069 (613M) [application/octet-stream]\n","Saving to: ‘gatk-4.4.0.0.zip’\n","\n","gatk-4.4.0.0.zip    100%[===================>] 612.70M  94.6MB/s    in 5.9s    \n","\n","2025-11-15 16:15:43 (104 MB/s) - ‘gatk-4.4.0.0.zip’ saved [642466069/642466069]\n","\n","Archive:  gatk-4.4.0.0.zip\n","   creating: gatk-4.4.0.0/\n","   creating: gatk-4.4.0.0/gatkdoc/\n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_ASEReadCounter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_replicates_CollectIndependentReplicateMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_dragstr_CalibrateDragstrModel.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ChromosomeCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_LibraryReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_DumpTabixIndex.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_FilterVariantTranches.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonChimericOriginalAlignmentReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_MeanQualityByCycle.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintReadsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordance.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CheckTerminatorBlock.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadLengthReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_Pileup.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_OverclippedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CleanSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_GtcToVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_CheckPileup.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_BwaMemIndexImageCreator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerMotifs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OrientationBiasReadCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_RenameSampleInVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_ReadAnonymizer.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityAvailableReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_CountNs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AssemblyComplexity.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectOxoGMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_VerifyIDIntensityContaminationMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotProperlyPairedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_FingerprintingDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ExcessiveEndClippedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaReferenceMaker.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_BarcodeMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_WellformedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AlleleFraction.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_QualityScoreDistributionSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerSampleHC.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_SampleReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_AnalyzeCovariates.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RMSMappingQuality.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$FirstOfPairReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_ConvertSequencingArtifactToOxoG.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountBasesSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountReadsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_Funcotator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectQualityYieldMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CreateReadCountPanelOfNormals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_CalcMetadataSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SecondOfPairReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_AlignmentAgreesWithHeaderReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_CollectDuplicateMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_StrandOddsRatio.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_filter_FilterVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage$WgsMetricsWithNonZeroCoverage.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_ParallelCopyGCSDirectoryIntoHDFSSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_IndelErrorMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_ShiftFasta.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_varianteval_VariantEval.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_GcBiasSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRrbsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RnaSeqMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MendelianViolations_FindMendelianViolations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSecondaryAlignmentReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_CountReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_FixVcfHeader.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_CollectHsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_AccumulateQualityYieldMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_gnarlyGenotyper_GnarlyGenotyper.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_JumpingLibraryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentEndReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_EstimateLibraryComplexity.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReferenceBases.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintReadsHeader.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ViewSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithExpectedAlleleFraction.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_ErrorMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CountVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_IndexFeatureFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$AllowAllReadsReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_LiftOverIntervalList.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MetricsReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotDuplicateReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBadGenomicKmersSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_OverlappingErrorMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fastq_BamToBfq.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddCommentsToBam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_QualByDepth.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_AnalyzeSaturationMutagenesis.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics$VariantCallingSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_GenotypeGVCFs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$PreAdapterSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_TandemRepeat.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysControlCodesSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_BuildBamIndex.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_VariantAnnotator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FilterSamReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_BaseQualityRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_MergePedIntoVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_Concordance.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_FragmentLengthReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_SplitNCigarReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PrimaryLineReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroFragmentLengthReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_BafRegressMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectMultipleMetricsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_reference_CompareReferences.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_StructuralVariationDiscoveryPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_EvaluateInfoFieldConcordance.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_CountBases.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandOddsRatio.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_SplitVcfs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/index.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SplitSamByLibrary.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FilterFuncotations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MendelianViolations_MendelianViolationMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CheckFingerprint.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantTrain.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$HasReadGroupReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CrosscheckFingerprints.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_coverage_DepthOfCoverage.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_JointGermlineCNVSegmentation.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_BaseErrorMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_PileupSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DetermineGermlineContigPloidy.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBreakpointEvidenceSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_UniqueAltReadCount.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ProperlyPairedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQuality.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$CigarContainsNoNOperator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotDenoisedCopyRatios.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_QualByDepth.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_featuremapping_FlowFeatureMapper.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_FlowBasedTPAttributeSymetricReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_VariantsToTable.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqFilterSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_VariantRecalibrator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectBaseDistributionByCycleSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_utils_ExtractOriginalAlignmentRecordsByNameSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantrecalling_HaplotypeBasedVariantRecaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_NormalizeFasta.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVConcordance.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_CycleSkipStatus.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_UnmarkDuplicates.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerIndelNuc.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_RMSMappingQuality.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics$PFFailSummaryMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantWriteTensors.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_SplitIntervals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaAlternateReferenceMaker.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountVariantsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_UpdateVCFSequenceDictionary.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallsToFastq.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamFormatConverter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_PostProcessReadsForRSEM.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_GeneExpressionEvaluation.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_ClusterCrosscheckMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_ClipReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_CollectSamErrorMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_ReadGroupHasFlowOrderReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_SoftClippedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_GetSampleName.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_CollectSVEvidence.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_MappingQualityRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_sv_SiteDepthtoBAF.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_IntervalOverlapReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerAlleleBySample.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroReferenceLengthAlignmentReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CalculateReadGroupChecksum.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_groundtruth_GroundTruthReadsBuilder.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_RampedHaplotypeCaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_HaplotypeFilteringAnnotation.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQuality.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SeqIsStoredReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SetNmMdAndUqTags.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_metrics_GcBiasMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CompareMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotateSegments.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_GatherBamFiles.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$BaitBiasSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CalculateFingerprintMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotatorDataSourceDownloader.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_ConvertHeaderlessHadoopBamShardToBam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildReferenceTaxonomy.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_FisherStrand.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_validation_CompareBaseQualities.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_replicates_IndependentReplicateMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_bwa_BwaSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupBlackListReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamComparisonMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FastqToSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectAlignmentSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_WgsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CollectIlluminaBasecallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_GcBiasDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SortSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithBamDepth.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamToFastq.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_CollectSequencingArtifactMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqScoreSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_RevertSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceContingencyMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GatherVcfs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_SplitReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PassesVendorQualityCheckReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_ExtractSequences.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_MarkDuplicatesWithMateCigar.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRnaSeqMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_GatherBQSRReports.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_CalculateContamination.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_MergeSamFiles.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_ApplyBQSR.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_GatherVcfsCloud.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_inference_CpxVariantReInterpreterSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_AnnotateIntervals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildKmers.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MakeSitesOnlyVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_IntervalListTools.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_InbreedingCoeff.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaPhasingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectInsertSizeMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ReadLengthEqualsCigarLengthReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CombineGenotypingArrayVcfs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandBiasBySample.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_BaitDesigner.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentDepthPerAlleleBySample.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_readorientation_LearnReadOrientationModel.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ValidateSamFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectReadCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_illumina_BpmToNormalizationManifestCsv.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_FlowBasedTPAttributeValidReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_InsertSizeMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_SortSamSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_LeftAlignIndels.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_FilterIntervals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityNotZeroReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_SortVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_GetNormalArtifactData.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FixMateInformation.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintVariantsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_HaplotypeCallerSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectJumpingLibraryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SplitSamByNumberOfReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_HsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_dragstr_ComposeSTRTableFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_MethylationTypeCaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectOxoGMetrics$CpcgMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateDifferentStrandReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_NonNFastaSize.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_filtering_FilterMutectCalls.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_FingerprintMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_BaseRecalibratorSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$BaitBiasDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CalculateMixingFractions.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_SelectVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_AccumulateVariantCallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallsToSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVCluster.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_FlagStat.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintDistantMates.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_FingerprintingSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_ScoreVariantAnnotations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PreprocessIntervals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_QualityScoreDistribution.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MappingQualityReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/gatkDoc.css  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_WellformedFlowBasedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectMultipleMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_CalculateGenotypePosteriors.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_genomicsdb_GenomicsDBImport.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MergeVcfs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectQualityYieldMetricsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformUnitReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNScoreVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_RevertOriginalBaseQualitiesAndAddMateCigar.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CreateBafRegressMetricsFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_IndelLength.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_ExtractSVEvidenceSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ValidateVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RawGtCount.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectAllelicCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_CreateHadoopBamSplittingIndex.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadTagValueFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysVariantCallingSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_LiftoverVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_HmerQualitySymetricReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BwaAndMarkDuplicatesPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddOATag.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MateDistantReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_HaplotypeCaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_AlignmentSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_ApplyBQSRSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CreateVerifyIDIntensityContaminationMetricsFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosition.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_Coverage.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentStartReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateOnSameContigOrNoMappedMateReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_ReadsPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadStrandFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_TargetedPcrMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ReplaceSamHeader.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CountFalsePositives.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateUnmappedAndUnmappedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVAnnotate.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_TargetMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddOrReplaceReadGroups.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_VcfFormatConverter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_VariantType.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_BamIndexStats.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaLaneMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CollectIlluminaLaneMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_CountBasesInReference.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_ClusteredCrosscheckMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CombineGVCFs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_JexlExpressionReadTagValueFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectGcBiasMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_LocalAssembler.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_sv_PrintReadCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_VcfToIntervalList.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SetNmAndUqTags.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBwaSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_TheoreticalSensitivityMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotModeledSegments.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_MarkIlluminaAdapters.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_SampleList.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_UpdateVcfSequenceDictionary.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MatchingBasesAndQualsReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_DiscoverVariantsFromContigAlignmentsSAMSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_GermlineCNVCaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_MeanQualityByCycleSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentLength.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CheckIlluminaDirectory.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadNameReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_CollectTargetedPcrMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectBaseDistributionByCycle.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_ApplyVQSR.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RrbsSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ReblockGVCF.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RrbsCpgDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_ErrorSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CallCopyRatioSegments.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_InbreedingCoeff.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_ScatterIntervalsByNs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectQualityYieldMetrics$QualityYieldMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_TrainVariantAnnotationsModel.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_MergeBamAlignment.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_ExtractVariantAnnotations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_FixMisencodedBaseQualityReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityZero.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_LeftAlignAndTrimVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_ModelSegments.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FeaturizedReadSets.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_BaseRecalibrator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_DuplicationMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_GatherTranches.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_RevertSamSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_ExtractIlluminaBarcodes.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_RevertBaseQualityScores.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics$PFFailDetailedMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_DownsampleSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ClippingRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_AmbiguousBaseReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_BedToIntervalList.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ExcessHet.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_RemoveNearbyIndels.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_filters_VariantFiltration.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_LikelihoodRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_validation_CompareDuplicatesSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRawWgsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_PossibleDeNovo.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_IndelClassify.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysVariantCallingDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_TransferReadTags.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectInsertSizeMetricsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQualityRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_SvDiscoverFromLocalAssemblyContigAlignmentsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DenoiseReadCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_IntervalListToBed.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerIndelLength.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CompareSAMs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics$VariantCallingDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_basicshortmutpileup_ValidateBasicSomaticShortMutations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OriginalAlignment.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BQSRPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_MarkDuplicates.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_UmiAwareMarkDuplicatesWithMateCigar.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_GcContent.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CreateSequenceDictionary.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_VcfToAdpc.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$GoodCigarReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_BaseDistributionByCycleMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_realignmentfilter_FilterAlignmentArtifacts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_SimpleMarkDuplicatesWithMateCigar.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_GenotypeSummaries.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_FlagStatSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_UmiMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ReorderSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRawWgsMetrics$RawWgsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AllelePseudoDepth.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PostprocessGermlineCNVCalls.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSupplementaryAlignmentReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_ReadPosRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_FifoBuffer.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_PositionBasedDownsampleSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_GetPileupSummaries.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_reference_CheckReferenceCompatibility.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PairedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$PreAdapterDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ReorderSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRawWgsMetrics$RawWgsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AllelePseudoDepth.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PostprocessGermlineCNVCalls.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSupplementaryAlignmentReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_BaseDistributionByCycleMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_GenotypeSummaries.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_FlagStatSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_SimpleMarkDuplicatesWithMateCigar.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_realignmentfilter_FilterAlignmentArtifacts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_UmiMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_reference_CheckReferenceCompatibility.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PairedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$PreAdapterDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_ReadPosRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_GetPileupSummaries.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_FifoBuffer.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_PositionBasedDownsampleSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_AmbiguousBaseReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_BedToIntervalList.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ExcessHet.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_RemoveNearbyIndels.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_filters_VariantFiltration.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRawWgsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_validation_CompareDuplicatesSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_LikelihoodRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_BaseRecalibrator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_DuplicationMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_ExtractIlluminaBarcodes.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_RevertBaseQualityScores.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_RevertSamSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_GatherTranches.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics$PFFailDetailedMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_DownsampleSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ClippingRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_basicshortmutpileup_ValidateBasicSomaticShortMutations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics$VariantCallingDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CompareSAMs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BQSRPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OriginalAlignment.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_UmiAwareMarkDuplicatesWithMateCigar.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_MarkDuplicates.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_VcfToAdpc.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$GoodCigarReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_GcContent.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CreateSequenceDictionary.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysVariantCallingDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_TransferReadTags.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_IndelClassify.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_PossibleDeNovo.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectInsertSizeMetricsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DenoiseReadCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQualityRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_SvDiscoverFromLocalAssemblyContigAlignmentsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerIndelLength.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_IntervalListToBed.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_ApplyVQSR.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RrbsSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ReblockGVCF.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_MeanQualityByCycleSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_DiscoverVariantsFromContigAlignmentsSAMSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_GermlineCNVCaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_UpdateVcfSequenceDictionary.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MatchingBasesAndQualsReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CheckIlluminaDirectory.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentLength.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_CollectTargetedPcrMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadNameReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectBaseDistributionByCycle.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_TrainVariantAnnotationsModel.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_FixMisencodedBaseQualityReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_ExtractVariantAnnotations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_MergeBamAlignment.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityZero.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FeaturizedReadSets.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_ModelSegments.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_LeftAlignAndTrimVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RrbsCpgDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_ErrorSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CallCopyRatioSegments.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_ScatterIntervalsByNs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_InbreedingCoeff.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectQualityYieldMetrics$QualityYieldMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaLaneMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_BamIndexStats.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_Mutect2.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CountFalsePositives.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateUnmappedAndUnmappedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_TargetMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVAnnotate.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_VariantType.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_VcfFormatConverter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddOrReplaceReadGroups.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBwaSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SetNmAndUqTags.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotModeledSegments.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_MarkIlluminaAdapters.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_TheoreticalSensitivityMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_SampleList.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CombineGVCFs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_ClusteredCrosscheckMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_CountBasesInReference.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CollectIlluminaLaneMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_LocalAssembler.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_sv_PrintReadCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectGcBiasMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_JexlExpressionReadTagValueFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_VcfToIntervalList.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_LiftoverVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysVariantCallingSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_BwaAndMarkDuplicatesPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_HmerQualitySymetricReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddOATag.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MateDistantReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_CreateHadoopBamSplittingIndex.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadTagValueFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosition.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_ReadsPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentStartReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateOnSameContigOrNoMappedMateReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_Coverage.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ReplaceSamHeader.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadStrandFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_TargetedPcrMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_HaplotypeCaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_AlignmentSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_ApplyBQSRSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CreateVerifyIDIntensityContaminationMetricsFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_CalculateGenotypePosteriors.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_genomicsdb_GenomicsDBImport.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MergeVcfs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectMultipleMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_WellformedFlowBasedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CreateBafRegressMetricsFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_IndelLength.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_ValidateVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RawGtCount.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_ExtractSVEvidenceSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectAllelicCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_PlatformUnitReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectQualityYieldMetricsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNScoreVariants.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_RevertOriginalBaseQualitiesAndAddMateCigar.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_SelectVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallsToSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_AccumulateVariantCallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_dragstr_ComposeSTRTableFile.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectJumpingLibraryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SplitSamByNumberOfReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_HsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MateDifferentStrandReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_NonNFastaSize.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectOxoGMetrics$CpcgMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_MethylationTypeCaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_BaseRecalibratorSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_filtering_FilterMutectCalls.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_FingerprintMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_CalculateMixingFractions.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$BaitBiasDetailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_FingerprintingSummaryMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_PreprocessIntervals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_scalable_ScoreVariantAnnotations.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MappingQualityReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_QualityScoreDistribution.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVCluster.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintDistantMates.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_FlagStat.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CombineGenotypingArrayVcfs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_BaitDesigner.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandBiasBySample.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MakeSitesOnlyVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildKmers.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_IntervalListTools.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaPhasingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_InbreedingCoeff.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ReadLengthEqualsCigarLengthReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectInsertSizeMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQualityRankSumTest.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_FilterIntervals.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_LeftAlignIndels.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityNotZeroReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FixMateInformation.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_GetNormalArtifactData.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_SortVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_HaplotypeCallerSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintVariantsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CollectReadCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ValidateSamFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_readorientation_LearnReadOrientationModel.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FragmentDepthPerAlleleBySample.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_FlowBasedTPAttributeValidReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_illumina_BpmToNormalizationManifestCsv.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_SortSamSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_InsertSizeMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_replicates_IndependentReplicateMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FastqToSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamComparisonMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_bwa_BwaSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadGroupBlackListReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_WgsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectAlignmentSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamToFastq.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SortSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithBamDepth.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_GcBiasDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_CollectIlluminaBasecallingMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_GatherBamFiles.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$BaitBiasSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotatorDataSourceDownloader.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CalculateFingerprintMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_FisherStrand.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqBuildReferenceTaxonomy.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_ConvertHeaderlessHadoopBamShardToBam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_validation_CompareBaseQualities.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_MarkDuplicatesWithMateCigar.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_MergeSamFiles.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_ApplyBQSR.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_contamination_CalculateContamination.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRnaSeqMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_GatherBQSRReports.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_GatherVcfsCloud.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_AnnotateIntervals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_discovery_inference_CpxVariantReInterpreterSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_CollectSequencingArtifactMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqScoreSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordanceContingencyMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_RevertSam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GatherVcfs.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_ExtractSequences.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PassesVendorQualityCheckReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_SplitReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_ReadGroupHasFlowOrderReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_SoftClippedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_CollectSamErrorMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_MappingQualityRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_GetSampleName.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_CollectSVEvidence.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_sv_SiteDepthtoBAF.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_PostProcessReadsForRSEM.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamFormatConverter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_GeneExpressionEvaluation.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_ClusterCrosscheckMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_ClipReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReadPosRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_BaseQuality.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SeqIsStoredReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SetNmMdAndUqTags.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_metrics_GcBiasMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CompareMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FuncotateSegments.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerAlleleBySample.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_IntervalOverlapReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CalculateReadGroupChecksum.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroReferenceLengthAlignmentReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_haplotypecaller_RampedHaplotypeCaller.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_groundtruth_GroundTruthReadsBuilder.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_HaplotypeFilteringAnnotation.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectBaseDistributionByCycleSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_VariantRecalibrator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqFilterSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantrecalling_HaplotypeBasedVariantRecaller.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_utils_ExtractOriginalAlignmentRecordsByNameSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_SVConcordance.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_reference_NormalizeFasta.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_plotting_PlotDenoisedCopyRatios.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_QualByDepth.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_featuremapping_FlowFeatureMapper.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_flow_FlowBasedTPAttributeSymetricReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_VariantsToTable.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics$PFFailSummaryMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_SplitIntervals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantWriteTensors.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaAlternateReferenceMaker.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_variantutils_UpdateVCFSequenceDictionary.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallsToFastq.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountVariantsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_CycleSkipStatus.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_UnmarkDuplicates.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerIndelNuc.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_RMSMappingQuality.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_CollectMultipleMetricsSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_reference_CompareReferences.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_StructuralVariationDiscoveryPipelineSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_StrandOddsRatio.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_CountBases.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_EvaluateInfoFieldConcordance.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_SplitNCigarReads.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$PrimaryLineReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonZeroFragmentLengthReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_BafRegressMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_sv_JointGermlineCNVSegmentation.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_coverage_DepthOfCoverage.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_DetermineGermlineContigPloidy.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBreakpointEvidenceSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_UniqueAltReadCount.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_PileupSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_BaseErrorMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ProperlyPairedReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$CigarContainsNoNOperator.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_MappingQuality.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_SplitVcfs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_FilterFuncotations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MendelianViolations_MendelianViolationMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SplitSamByLibrary.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CrosscheckFingerprints.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$HasReadGroupReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fingerprint_CheckFingerprint.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_CNNVariantTrain.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_CollectVariantCallingMetrics$VariantCallingSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_GenotypeGVCFs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_CollectArraysVariantCallingMetrics$ArraysControlCodesSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_TandemRepeat.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_SequencingArtifactMetrics$PreAdapterSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_FindBadGenomicKmersSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_OverlappingErrorMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_fastq_BamToBfq.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_AddCommentsToBam.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_AnalyzeSaturationMutagenesis.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_QualByDepth.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_VariantAnnotator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_FilterSamReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_BaseQualityRankSumTest.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_MergePedIntoVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_Concordance.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_FragmentLengthReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_BuildBamIndex.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_RnaSeqMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectRrbsMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotSecondaryAlignmentReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_MendelianViolations_FindMendelianViolations.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_directed_CollectHsMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_FixVcfHeader.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_CountReads.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage$WgsMetricsWithNonZeroCoverage.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_filter_FilterVcf.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_ParallelCopyGCSDirectoryIntoHDFSSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_IndelErrorMetric.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_ShiftFasta.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_varianteval_VariantEval.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_GcBiasSummaryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ReferenceBases.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_PrintReadsHeader.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_ViewSam.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_validation_AnnotateVcfWithExpectedAlleleFraction.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_IndexFeatureFile.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$AllowAllReadsReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_CountVariants.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_SamErrorMetric_ErrorMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_LiftOverIntervalList.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_MetricsReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotDuplicateReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_util_AccumulateQualityYieldMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_IlluminaBasecallingMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_JumpingLibraryMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_gnarlyGenotyper_GnarlyGenotyper.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_EstimateLibraryComplexity.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$ValidAlignmentEndReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_FingerprintingDetailMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ExcessiveEndClippedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_BarcodeMetric.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_fasta_FastaReferenceMaker.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_SampleReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_metrics_QualityScoreDistributionSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_DepthPerSampleHC.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_WellformedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AlleleFraction.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_artifacts_ConvertSequencingArtifactToOxoG.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$FirstOfPairReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_RMSMappingQuality.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_bqsr_AnalyzeCovariates.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_OrientationBiasReadCounts.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_AssemblyComplexity.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_CountNs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_RenameSampleInVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_ReadAnonymizer.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$MappingQualityAvailableReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectOxoGMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NotProperlyPairedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_VerifyIDIntensityContaminationMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_copynumber_CreateReadCountPanelOfNormals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_sv_evidence_CalcMetadataSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pathseq_PathSeqPipelineSpark.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_markduplicates_CollectDuplicateMetrics.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_AlignmentAgreesWithHeaderReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$SecondOfPairReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_allelespecific_AS_StrandOddsRatio.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_funcotator_Funcotator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountReadsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_CountBasesSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectQualityYieldMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_MeanQualityByCycle.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_illumina_quality_CollectHiSeqXPfFailMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_vcf_GenotypeConcordance.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_spark_pipelines_PrintReadsSpark.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_replicates_CollectIndependentReplicateMetrics.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_rnaseq_ASEReadCounter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_ChromosomeCounts.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_dragstr_CalibrateDragstrModel.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_DumpTabixIndex.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_vqsr_FilterVariantTranches.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_LibraryReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadFilterLibrary$NonChimericOriginalAlignmentReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_analysis_CollectWgsMetricsWithNonZeroCoverage.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_CheckPileup.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_arrays_GtcToVcf.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_annotator_flow_HmerMotifs.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_BwaMemIndexImageCreator.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CheckTerminatorBlock.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_ReadLengthReadFilter.html  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_engine_filters_OverclippedReadFilter.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/org_broadinstitute_hellbender_tools_walkers_qc_Pileup.json  \n","  inflating: gatk-4.4.0.0/gatkdoc/picard_sam_CleanSam.html  \n","  inflating: gatk-4.4.0.0/gatkPythonPackageArchive.zip  \n","  inflating: gatk-4.4.0.0/gatk-completion.sh  \n","  inflating: gatk-4.4.0.0/gatk-package-4.4.0.0-spark.jar  \n","  inflating: gatk-4.4.0.0/gatkcondaenv.yml  \n","  inflating: gatk-4.4.0.0/README.md  \n","   creating: gatk-4.4.0.0/scripts/\n","   creating: gatk-4.4.0.0/scripts/sv/\n","  inflating: gatk-4.4.0.0/scripts/sv/create_cluster.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/delete_cluster.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/sanity_checks.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/manage_sv_pipeline.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/run_whole_pipeline.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/copy_sv_results.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/default_init.sh  \n","   creating: gatk-4.4.0.0/scripts/sv/stepByStep/\n","  inflating: gatk-4.4.0.0/scripts/sv/stepByStep/scanBam.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/stepByStep/discoverVariants.sh  \n","  inflating: gatk-4.4.0.0/scripts/sv/stepByStep/svDiscover.sh  \n","   creating: gatk-4.4.0.0/scripts/cnv_wdl/\n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/cnv_common_tasks.wdl  \n","   creating: gatk-4.4.0.0/scripts/cnv_wdl/germline/\n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/cnv_germline_cohort_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/README.md  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/cnv_germline_case_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/Structs.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/cnv_germline_case_scattered_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/germline/joint_call_exome_cnvs.wdl  \n","   creating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/\n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/cnv_somatic_oncotator_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/README.md  \n","  inflating: gatk-4.4.0.0/scripts/cnv_wdl/somatic/cnv_somatic_funcotate_seg_workflow.wdl  \n","  inflating: gatk-4.4.0.0/scripts/dataproc-cluster-ui  \n","   creating: gatk-4.4.0.0/scripts/mutect2_wdl/\n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect2.wdl  \n","   creating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect_resources_json/\n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect_resources_json/mutect_resources_process_gnomAD_2.1.json  \n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/README.md  \n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect2_pon.wdl  \n","  inflating: gatk-4.4.0.0/scripts/mutect2_wdl/mutect_resources.wdl  \n","  inflating: gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar  \n","  inflating: gatk-4.4.0.0/gatk       \n","  inflating: gatk-4.4.0.0/GATKConfig.EXAMPLE.properties  \n"]}]},{"cell_type":"code","source":["!cd /content/CTC360/ref\n","\n","# Download only chromosome 12 FASTA (hg38)\n","!wget -c https://hgdownload.soe.ucsc.edu/goldenPath/hg38/chromosomes/chr12.fa.gz\n","!gunzip chr12.fa.gz\n","!mv chr12.fa hg38.fa\n","\n","# Index reference\n","!bwa index hg38.fa\n","!samtools faidx hg38.fa\n","!gatk-4.4.0.0/gatk CreateSequenceDictionary -R hg38.fa\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"id":"50MHbYgjTs5u","executionInfo":{"status":"ok","timestamp":1763223636530,"user_tz":300,"elapsed":157350,"user":{"displayName":"Aayudh Das","userId":"08660880840353696711"}},"outputId":"54268635-dd39-412c-eeea-6bbca25fef3e"},"execution_count":14,"outputs":[{"output_type":"stream","name":"stdout","text":["--2025-11-15 16:17:58--  https://hgdownload.soe.ucsc.edu/goldenPath/hg38/chromosomes/chr12.fa.gz\n","Resolving hgdownload.soe.ucsc.edu (hgdownload.soe.ucsc.edu)... 128.114.119.163\n","Connecting to hgdownload.soe.ucsc.edu (hgdownload.soe.ucsc.edu)|128.114.119.163|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 42940449 (41M) [application/x-gzip]\n","Saving to: ‘chr12.fa.gz’\n","\n","chr12.fa.gz         100%[===================>]  40.95M  26.8MB/s    in 1.5s    \n","\n","2025-11-15 16:18:00 (26.8 MB/s) - ‘chr12.fa.gz’ saved [42940449/42940449]\n","\n","[bwa_index] Pack FASTA... 1.07 sec\n","[bwa_index] Construct BWT for the packed sequence...\n","[BWTIncCreate] textLength=266550618, availableWord=30755168\n","[BWTIncConstructFromPacked] 10 iterations done. 50732074 characters processed.\n","[BWTIncConstructFromPacked] 20 iterations done. 93723498 characters processed.\n","[BWTIncConstructFromPacked] 30 iterations done. 131930362 characters processed.\n","[BWTIncConstructFromPacked] 40 iterations done. 165884698 characters processed.\n","[BWTIncConstructFromPacked] 50 iterations done. 196059354 characters processed.\n","[BWTIncConstructFromPacked] 60 iterations done. 222874650 characters processed.\n","[BWTIncConstructFromPacked] 70 iterations done. 246704106 characters processed.\n","[BWTIncConstructFromPacked] 80 iterations done. 266550618 characters processed.\n","[bwt_gen] Finished constructing BWT in 80 iterations.\n","[bwa_index] 98.89 seconds elapse.\n","[bwa_index] Update BWT... 0.75 sec\n","[bwa_index] Pack forward-only FASTA... 0.59 sec\n","[bwa_index] Construct SA from BWT and Occ... 42.95 sec\n","[main] Version: 0.7.17-r1188\n","[main] CMD: bwa index hg38.fa\n","[main] Real time: 147.284 sec; CPU: 144.259 sec\n","Using GATK jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar\n","Running:\n","    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar CreateSequenceDictionary -R hg38.fa\n","INFO\t2025-11-15 16:20:33\tCreateSequenceDictionary\tOutput dictionary will be written in hg38.dict\n","16:20:33.975 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/content/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so\n","[Sat Nov 15 16:20:34 UTC 2025] CreateSequenceDictionary --REFERENCE hg38.fa --TRUNCATE_NAMES_AT_WHITESPACE true --NUM_SEQUENCES 2147483647 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX false --CREATE_MD5_FILE false --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\n","[Sat Nov 15 16:20:34 UTC 2025] Executing as root@4451a6451ba3 on Linux 6.6.105+ amd64; OpenJDK 64-Bit Server VM 17.0.16+8-Ubuntu-0ubuntu122.04.1; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.4.0.0\n","[Sat Nov 15 16:20:35 UTC 2025] picard.sam.CreateSequenceDictionary done. Elapsed time: 0.03 minutes.\n","Runtime.totalMemory()=239075328\n","Tool returned:\n","0\n"]}]},{"cell_type":"code","source":["!cd /content/CTC360/fastq\n","\n","# Download SRA files\n","!prefetch SRR13973704\n","!prefetch SRR13973878\n","\n","# Convert to FASTQ\n","!fasterq-dump SRR13973704 -O .\n","!fasterq-dump SRR13973878 -O .\n","\n","# Compress\n","!gzip SRR13973704_1.fastq SRR13973704_2.fastq\n","!gzip SRR13973878_1.fastq SRR13973878_2.fastq\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"collapsed":true,"id":"MNUKn5GMUat_","outputId":"8ce60d18-a0ea-4c26-dffe-153934e97e82"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["\n","2025-11-15T16:20:52 prefetch.2.11.3: Current preference is set to retrieve SRA Normalized Format files with full base quality scores.\n","2025-11-15T16:20:52 prefetch.2.11.3: 1) Downloading 'SRR13973704'...\n","2025-11-15T16:20:52 prefetch.2.11.3: SRA Normalized Format file is being retrieved, if this is different from your preference, it may be due to current file availability.\n","2025-11-15T16:20:52 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:14 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:22 prefetch.2.11.3:  'SRR13973704' is valid\n","2025-11-15T16:21:22 prefetch.2.11.3: 1) 'SRR13973704' was downloaded successfully\n","2025-11-15T16:21:27 prefetch.2.11.3: 'SRR13973704' has 25 unresolved dependencies\n","2025-11-15T16:21:27 prefetch.2.11.3: 2) Downloading 'ncbi-acc:CM000663.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:27 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:28 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:28 prefetch.2.11.3: 2) 'ncbi-acc:CM000663.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:28 prefetch.2.11.3: 3) Downloading 'ncbi-acc:CM000664.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:28 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:29 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:29 prefetch.2.11.3: 3) 'ncbi-acc:CM000664.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:29 prefetch.2.11.3: 4) Downloading 'ncbi-acc:CM000665.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:29 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:29 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:29 prefetch.2.11.3: 4) 'ncbi-acc:CM000665.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:29 prefetch.2.11.3: 5) Downloading 'ncbi-acc:CM000666.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:29 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:30 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:30 prefetch.2.11.3: 5) 'ncbi-acc:CM000666.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:30 prefetch.2.11.3: 6) Downloading 'ncbi-acc:CM000667.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:30 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:31 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:31 prefetch.2.11.3: 6) 'ncbi-acc:CM000667.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:31 prefetch.2.11.3: 7) Downloading 'ncbi-acc:CM000668.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:31 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:31 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:31 prefetch.2.11.3: 7) 'ncbi-acc:CM000668.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:31 prefetch.2.11.3: 8) Downloading 'ncbi-acc:CM000669.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:31 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:32 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:32 prefetch.2.11.3: 8) 'ncbi-acc:CM000669.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:32 prefetch.2.11.3: 9) Downloading 'ncbi-acc:CM000670.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:32 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:32 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:32 prefetch.2.11.3: 9) 'ncbi-acc:CM000670.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:32 prefetch.2.11.3: 10) Downloading 'ncbi-acc:CM000671.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:32 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:33 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:33 prefetch.2.11.3: 10) 'ncbi-acc:CM000671.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:33 prefetch.2.11.3: 11) Downloading 'ncbi-acc:CM000672.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:33 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:33 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:33 prefetch.2.11.3: 11) 'ncbi-acc:CM000672.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:33 prefetch.2.11.3: 12) Downloading 'ncbi-acc:CM000673.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:33 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:33 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:33 prefetch.2.11.3: 12) 'ncbi-acc:CM000673.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:33 prefetch.2.11.3: 13) Downloading 'ncbi-acc:CM000674.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:33 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:34 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:34 prefetch.2.11.3: 13) 'ncbi-acc:CM000674.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:34 prefetch.2.11.3: 14) Downloading 'ncbi-acc:CM000675.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:34 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:34 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:34 prefetch.2.11.3: 14) 'ncbi-acc:CM000675.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:34 prefetch.2.11.3: 15) Downloading 'ncbi-acc:CM000676.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:34 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:35 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:35 prefetch.2.11.3: 15) 'ncbi-acc:CM000676.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:35 prefetch.2.11.3: 16) Downloading 'ncbi-acc:CM000677.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:35 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:35 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:35 prefetch.2.11.3: 16) 'ncbi-acc:CM000677.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:35 prefetch.2.11.3: 17) Downloading 'ncbi-acc:CM000678.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:35 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:35 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:35 prefetch.2.11.3: 17) 'ncbi-acc:CM000678.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:35 prefetch.2.11.3: 18) Downloading 'ncbi-acc:CM000679.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:35 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:36 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:36 prefetch.2.11.3: 18) 'ncbi-acc:CM000679.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:36 prefetch.2.11.3: 19) Downloading 'ncbi-acc:CM000680.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:36 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:36 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:36 prefetch.2.11.3: 19) 'ncbi-acc:CM000680.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:36 prefetch.2.11.3: 20) Downloading 'ncbi-acc:CM000681.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:36 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:36 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:36 prefetch.2.11.3: 20) 'ncbi-acc:CM000681.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:36 prefetch.2.11.3: 21) Downloading 'ncbi-acc:CM000682.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:36 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:37 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:37 prefetch.2.11.3: 21) 'ncbi-acc:CM000682.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:37 prefetch.2.11.3: 22) Downloading 'ncbi-acc:CM000683.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:37 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:37 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:37 prefetch.2.11.3: 22) 'ncbi-acc:CM000683.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:37 prefetch.2.11.3: 23) Downloading 'ncbi-acc:CM000684.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:37 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:37 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:37 prefetch.2.11.3: 23) 'ncbi-acc:CM000684.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:37 prefetch.2.11.3: 24) Downloading 'ncbi-acc:CM000685.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:37 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:38 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:38 prefetch.2.11.3: 24) 'ncbi-acc:CM000685.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:38 prefetch.2.11.3: 25) Downloading 'ncbi-acc:CM000686.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:38 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:38 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:38 prefetch.2.11.3: 25) 'ncbi-acc:CM000686.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:38 prefetch.2.11.3: 26) Downloading 'ncbi-acc:NC_012920.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:38 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:38 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:38 prefetch.2.11.3: 26) 'ncbi-acc:NC_012920.1?vdb-ctx=refseq' was downloaded successfully\n","\n","2025-11-15T16:21:38 prefetch.2.11.3: Current preference is set to retrieve SRA Normalized Format files with full base quality scores.\n","2025-11-15T16:21:38 prefetch.2.11.3: 1) Downloading 'SRR13973878'...\n","2025-11-15T16:21:38 prefetch.2.11.3: SRA Normalized Format file is being retrieved, if this is different from your preference, it may be due to current file availability.\n","2025-11-15T16:21:38 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:42 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:43 prefetch.2.11.3:  'SRR13973878' is valid\n","2025-11-15T16:21:43 prefetch.2.11.3: 1) 'SRR13973878' was downloaded successfully\n","2025-11-15T16:21:45 prefetch.2.11.3: 'SRR13973878' has 25 unresolved dependencies\n","2025-11-15T16:21:45 prefetch.2.11.3: 2) Downloading 'ncbi-acc:CM000663.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:45 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:46 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:46 prefetch.2.11.3: 2) 'ncbi-acc:CM000663.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:46 prefetch.2.11.3: 3) Downloading 'ncbi-acc:CM000664.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:46 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:47 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:47 prefetch.2.11.3: 3) 'ncbi-acc:CM000664.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:47 prefetch.2.11.3: 4) Downloading 'ncbi-acc:CM000665.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:47 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:47 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:47 prefetch.2.11.3: 4) 'ncbi-acc:CM000665.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:47 prefetch.2.11.3: 5) Downloading 'ncbi-acc:CM000666.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:47 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:48 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:48 prefetch.2.11.3: 5) 'ncbi-acc:CM000666.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:48 prefetch.2.11.3: 6) Downloading 'ncbi-acc:CM000667.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:48 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:48 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:48 prefetch.2.11.3: 6) 'ncbi-acc:CM000667.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:48 prefetch.2.11.3: 7) Downloading 'ncbi-acc:CM000668.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:48 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:49 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:49 prefetch.2.11.3: 7) 'ncbi-acc:CM000668.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:49 prefetch.2.11.3: 8) Downloading 'ncbi-acc:CM000669.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:49 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:50 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:50 prefetch.2.11.3: 8) 'ncbi-acc:CM000669.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:50 prefetch.2.11.3: 9) Downloading 'ncbi-acc:CM000670.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:50 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:50 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:50 prefetch.2.11.3: 9) 'ncbi-acc:CM000670.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:50 prefetch.2.11.3: 10) Downloading 'ncbi-acc:CM000671.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:50 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:51 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:51 prefetch.2.11.3: 10) 'ncbi-acc:CM000671.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:51 prefetch.2.11.3: 11) Downloading 'ncbi-acc:CM000672.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:51 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:51 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:51 prefetch.2.11.3: 11) 'ncbi-acc:CM000672.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:51 prefetch.2.11.3: 12) Downloading 'ncbi-acc:CM000673.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:51 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:51 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:51 prefetch.2.11.3: 12) 'ncbi-acc:CM000673.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:51 prefetch.2.11.3: 13) Downloading 'ncbi-acc:CM000674.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:51 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:52 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:52 prefetch.2.11.3: 13) 'ncbi-acc:CM000674.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:52 prefetch.2.11.3: 14) Downloading 'ncbi-acc:CM000675.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:52 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:52 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:52 prefetch.2.11.3: 14) 'ncbi-acc:CM000675.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:52 prefetch.2.11.3: 15) Downloading 'ncbi-acc:CM000676.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:52 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:53 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:53 prefetch.2.11.3: 15) 'ncbi-acc:CM000676.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:53 prefetch.2.11.3: 16) Downloading 'ncbi-acc:CM000677.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:53 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:53 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:53 prefetch.2.11.3: 16) 'ncbi-acc:CM000677.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:53 prefetch.2.11.3: 17) Downloading 'ncbi-acc:CM000678.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:53 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:53 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:53 prefetch.2.11.3: 17) 'ncbi-acc:CM000678.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:53 prefetch.2.11.3: 18) Downloading 'ncbi-acc:CM000679.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:53 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:54 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:54 prefetch.2.11.3: 18) 'ncbi-acc:CM000679.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:54 prefetch.2.11.3: 19) Downloading 'ncbi-acc:CM000680.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:54 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:54 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:54 prefetch.2.11.3: 19) 'ncbi-acc:CM000680.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:54 prefetch.2.11.3: 20) Downloading 'ncbi-acc:CM000681.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:54 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:55 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:55 prefetch.2.11.3: 20) 'ncbi-acc:CM000681.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:55 prefetch.2.11.3: 21) Downloading 'ncbi-acc:CM000682.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:55 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:55 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:55 prefetch.2.11.3: 21) 'ncbi-acc:CM000682.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:55 prefetch.2.11.3: 22) Downloading 'ncbi-acc:CM000683.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:55 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:55 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:55 prefetch.2.11.3: 22) 'ncbi-acc:CM000683.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:55 prefetch.2.11.3: 23) Downloading 'ncbi-acc:CM000684.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:55 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:56 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:56 prefetch.2.11.3: 23) 'ncbi-acc:CM000684.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:56 prefetch.2.11.3: 24) Downloading 'ncbi-acc:CM000685.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:56 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:56 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:56 prefetch.2.11.3: 24) 'ncbi-acc:CM000685.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:56 prefetch.2.11.3: 25) Downloading 'ncbi-acc:CM000686.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:56 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:56 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:56 prefetch.2.11.3: 25) 'ncbi-acc:CM000686.1?vdb-ctx=refseq' was downloaded successfully\n","2025-11-15T16:21:56 prefetch.2.11.3: 26) Downloading 'ncbi-acc:NC_012920.1?vdb-ctx=refseq'...\n","2025-11-15T16:21:56 prefetch.2.11.3:  Downloading via HTTPS...\n","2025-11-15T16:21:57 prefetch.2.11.3:  HTTPS download succeed\n","2025-11-15T16:21:57 prefetch.2.11.3: 26) 'ncbi-acc:NC_012920.1?vdb-ctx=refseq' was downloaded successfully\n","spots read      : 8,159,199\n","reads read      : 16,318,398\n","reads written   : 16,218,868\n","spots read      : 1,548,246\n","reads read      : 3,096,492\n","reads written   : 3,028,842\n"]}]}]}